controllers/DefaultController.php

```diff
@@ -391,5 +396,26 @@ public function actionProfilePicture($id)
 		}
 		header('Content-Type:'.$picture['mimetype']);
 		echo $picture['picture'];
-	}
+	} 
+    
+    public function addUserToNotifications($userId)
+    {
+        if (class_exists('SubscribeBehavior')) {
+            $this->subscribeUser($userId, 'registerUser', 'Nowa rejestracja', 'user_' . $userId);
+            if (isset(Yii::app()->notificationsQueue)) {
+                $notificationsQueue = Yii::app()->notificationsQueue;
+                $queueId = (isset($notificationsQueue->id)) ? $notificationsQueue->id : 'notificationsQueue';
+                $label = (isset($notificationsQueue->label)) ? $notificationsQueue->label : 'Powiadomienia';
```

Comment:
Besides being too specific for the whole module, this method uses untranslated strings.